app/presenters/social_relay_presenter.rb

```diff
@@ -0,0 +1,27 @@
+class SocialRelayPresenter
+
+  def as_json options={}
+    {
+      "subscribe" => AppConfig.relay.inbound.subscribe,
+      "scope"     => AppConfig.relay.inbound.scope,
+      "tags"      => tags
+    }
+  end
+
+  def tags
+    unless AppConfig.relay.inbound.scope == "tags"
+      # No need to list tags
+      return []
+    end
+    tags = []
+    unless AppConfig.relay.inbound.pod_tags.blank?
+      tags = AppConfig.relay.inbound.pod_tags.split(",")
+    end
```

Comment:
That doesn't work because `pod_tags` can be empty but still `scope` can be tags and `include_user_tags` can be `true` in which case user tags could be returned, not `[]`. I'll change it a bit, see what you think after next push :)